hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/InboundResponseHandlerSupplierTest.java

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
Can you put a similar test for client implementation since I do not see any test added for testing client side changes?

Reply:
I doubt this is doable due to the untestability of the client code.